src/mixins/interfaces.ts

```diff
@@ -11,7 +11,7 @@ export interface Child extends Renderable, Destroyable {
 	/**
 	 * A reference to the widget's parent
 	 */
-	parent: Parent | null;
+	parent?: Parent | null;
```

Comment:
It is no longer mandatory as it has been removed from the new `Widget` interface